Makefile.org

```diff
@@ -685,7 +685,7 @@ install_docs:
 		(cd `$(PERL) util/dirname.pl $$i`; \
 		pod2man \
 			--section=$$sec --center=OpenSSL \
-			--release=$(VERSION) `basename $$i`) \
+			--release="OpenSSL $(VERSION)" `basename $$i`) \
```

Comment:
Why?  You already have the name in the header of the page...  Frankly, I think it's preferable that the release really just be the version number